.github/workflows/validate-data.yaml

```
@@ -1,9 +1,8 @@
 name: Validate Data
 
 on:
-  push:
-  pull_request:
-
+  push: null
+  pull_request: null
```

```suggestion
  push:
  pull_request:
```

The `null` isn't needed and isn't in other starter workflows, or in our docs so let's remove that.